src/odm_sharing/tools/share.py

```diff
@@ -156,6 +155,16 @@ def infer_outfmt(path: str) -> OutFmt:
         return OutFmt.EXCEL
 
 
+def get_input_name(input: str) -> str:
```

Comment:
Rename to `get_input_name_for_output_filename`? Since the return of the function should only be used for that context.

Reply:
I would prefer to not include what the return value is going to be used for in the function name. Isn't it clear as is? It's the name of the input, like the function implies:

- mydir/myfile.xlsx -> myfile
- myfile.csv -> ''

Well, I see the issue now. The CSV filename which turns into an empty string, which makes this function's use case specific to the output filename. What about renaming:
- `get_input_name` -> `get_output_prefix_from_input`
- `input_name` -> `output_prefix`

?